modules/quickstart-public-cloud/pages/clients.adoc

```diff
@@ -3,24 +3,14 @@
 
 When you have your {productname} Server set up, you are ready to start registering clients.
 
-
-
-.Procedure: Registering Clients with the {webui}
-
-. In the {productname} {webui}, navigate to menu:Systems[Bootstrapping].
 In the {productname} {webui}, navigate to menu:Systems[Bootstrapping], then fill in the ``Host``, ``SSH Port``, ``User`` and ``Password`` fields.
-
 Make sure you use stable FQDNs for the ``Host`` field, or {productname} will not be able to find your host when your Public Cloud gives you a different short-lived FQDNS.
 
-Also, please note public cloud images usually do not allow SSH login with user and password but only SSH certificate login. If you want to use the bootstrap from the UI, you will need to enable SSH login with user and password:
-<describe the required steps>
-
-{productname} will support bootstrapping from the UI with SSH certificates in a future release.
-
-Alternatively, you can use a bootstrap script.
-
-When the bootstrap process has completed, your client will be listed at menu:Systems[System List].
+Public cloud images usually do not allow SSH login with username and password, but only SSH with a certificate.
```

Comment:
```suggestion
Public cloud images usually do not allow SSH login with user and password, but only SSH with a certificate.
```
At other places nearby we seem to use just "user"

Reply:
but a user is a person, whereas a username is a piece of information that can be entered into a field?